### DISCUSSION TITLE
LTSP.CONF set keyboard layout

### DISCUSSION THREAD

**Discussion by Soylor** [ltsp/ltsp#788] [Q&A] [answered]
I'm trying to change client keyboard layout using
XKBLAYOUT = es

It's not working.
Any help?
Thanks.

**Comment by alkisg** [comment, marked as answer]
LTSP doesn't provide any options to set the keyboard layout. You need to set it using your distribution tools. Usually, the following should work:


```
sudo dpkg-reconfigure keyboard-configuration
```

**Reply by Soylor** [reply]
Perfect. I Did it and configured system wide.
Thank you.
